### PR TITLE
dplyr translations of the sql

### PR DESCRIPTION
**Pull request by jonkeane**

### CODE REVIEW

File: query-daylight-dplyr.Rmd [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+return only the row for the day with the max daylight in each month
+
+doesn't work; window expressions not allowed in WHERE clause
+```{sql, connection=con, eval=FALSE}
+SELECT month, day, light
+  FROM daylight
+ 	WHERE light = MAX(light) OVER(PARTITION BY month);
+```
+
+workaround: put it in the SELECT list
```
Comment (ianmcook) [inline query-daylight-dplyr.Rmd:60]: Can we delete this part? I think it's relevant to SQL but not to dplyr
```suggestion
```

Comment (ianmcook) [reply query-daylight-dplyr.Rmd:60]: OTOH maybe it's best to preserve the parallelism of this and the SQL version. What do you think @jonkeane ?

Comment (jonkeane) [reply query-daylight-dplyr.Rmd:60]: I like the parallelism, though maybe it would be better to make that explicit?

Comment (ianmcook) [reply query-daylight-dplyr.Rmd:60]: For now I'm going to merge this into a new branch of this repo named `r`. Later maybe we can come back and tidy up everything and merge together the branches.

**Pull request merged**